### ISSUE TITLE
Native Support for Postgres JSONB Functions

### ISSUE THREAD

**Issue by ajhollowayvrm** [prisma/prisma#16125]
## Problem

Dealing with raw JSONB with native Prisma can be a pain without using raw queries because of the lack of JSONB/JSON functions.

## Suggested solution

Implement JSONB/JSON functions natively.

## Additional context

[Postgres JSON Functions](https://www.postgresql.org/docs/current/functions-json.html)

**Comment by SevInf** [comment]
Hi @ajhollowayvrm.
Have you seen our [docs on json column filtering](https://www.prisma.io/docs/concepts/components/prisma-client/working-with-fields/working-with-json-fields)? Is there anything specific that is missing?

**Comment by ajhollowayvrm** [comment]
@SevInf On the topic of filters, I really wish this functionality was extended to Postgres (https://www.prisma.io/docs/concepts/components/prisma-client/working-with-fields/working-with-json-fields#filtering-on-object-key-value-inside-array). 

I'm thinking specifically of other operations such as writing to a JSONB array column with JSONB_Append.

**Comment by Sr-santi** [comment]
yeah, I agree with @ajhollowayvrm, it's supported in SQL but not in Postgres 😭.
CC @SevInf

**Comment by devtrix** [comment]
So, where is this issue now? Its been 6 months and any updates?

**Comment by PrerakGada** [comment]
Hello,
Are there any updates on this feature?
Is there a Prisma alternative to query through JSONB Fields like the Aggregation in MongoDB?
